src/v3/src/transformer/profile/transformEnrollProfile.ts

```diff
@@ -45,6 +54,13 @@ export const transformEnrollProfile: IdxStepTransformer = ({ transaction, formBa
         autocomplete: 'new-password',
       },
     };
+    // @ts-ignore expose type from auth-js
+    const passwordErrors = passwordElement.options.inputMeta.messages?.value;
+    if (passwordErrors?.length) {
+      const messages = updatePasswordRequirementsNotMetMessage(passwordErrors);
+      // @ts-ignore expose type from auth-js
+      passwordElement.options.inputMeta.messages.value = messages;
```

Comment:
Is there a JIRA we can attach? If not, can we file one for the SDK team and add it here?